packages/styles/scss/theme/_foundation.scss

```diff
@@ -61,6 +63,36 @@
 
   --ilo-color-purple: rgba(150, 10, 85, 1);
 
+  --ilo-color-notification-icon: var(--ilo-color-white);
```

Comment:
If it's the same color, we should use that token directly instead of redeclaring it like this right? @GGKapanadze